tiledb/sm/array/array.h

```diff
@@ -341,8 +347,14 @@ class Array {
   /*         PRIVATE ATTRIBUTES        */
   /* ********************************* */
 
-  /** The array schema. */
-  ArraySchema* array_schema_;
+  /** The latest array schema. */
+  ArraySchema* array_schema_latest_;
+
+  /**
+   * A map of all array_schemas_all
```

Comment:
Not very clear, but we can fix in a later PR. I am assuming that this is a map from the array schema file name to the array schema object.

Reply:
Find and replace got me, should say "map of all array schemas"